code/game/objects/items/weapons/chewables.dm

```diff
@@ -84,28 +84,28 @@ obj/item/clothing/mask/chewable/Destroy()
 	desc = "A disgusting spitwad."
 	icon = 'icons/obj/clothing/masks.dmi'
 	icon_state = "spit-chew"
+	slot_flags = SLOT_EARS | SLOT_MASK
 
-/obj/item/clothing/mask/chewable/proc/spitout(var/mob/user, var/no_message)
+/obj/item/clothing/mask/chewable/proc/spitout(var/transfer_color = 1, var/no_message = 0, var/mob/living/M)
```

Comment:
M doesn't need to be an argument, you can just define it on line 99 as it was previously.

Reply:
i like this better as it's in line with cigarette code

Reply:
No but that's bad programming.

`M` should not be a parameter if you don't care about it and just write over it.